virttest/qemu_vm.py

```diff
@@ -2083,9 +2083,13 @@ def add_disable_modern(devices, dev, dev_type):
                               "virtio-serial-pci", "virtio-rng-pci"]
         for device in devices:
             dev_type = device.get_param("driver")
+            set_disable_legacy = params.get("set_disable_legacy", "False")
+            set_disable_modern = params.get("set_disable_modern", "False")
             if dev_type in virtio_pci_devices:
-                add_disable_legacy(devices, device, dev_type)
-                add_disable_modern(devices, device, dev_type)
+                if set_disable_legacy == "True":
```

Comment:
"== True" looks strange. And I don't find any codes like that.
How about " set_disable_legacy = eval(params.get("set_disable_legacy", "False"))"?
and "if  set_disable_legacy: "